src/components/application_manager/src/commands/mobile/set_global_properties_request.cc

```diff
@@ -135,7 +135,7 @@ void SetGlobalPropertiesRequest::Run() {
   }
 
   if (is_vr_help_title_present && is_vr_help_present) {
-    LOG4CXX_DEBUG(logger_, "VRHelp params presents");
+    LOG4CXX_DEBUG(logger_, "VRHelp params are present");
```

Comment:
@LuxoftAKutsan Honestly forgot for this rule, splitted in separate commit 6ccb49b